02_combine_counts/combine_counts/combine_counts.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import os
+import sys
+
+import pandas as pd
+
+
+def merge_counts(count_files):
+    base_names = [os.path.basename(file_name) for file_name in count_files]
+    return [
+        pd.read_csv(
+            file_name,
+            sep="\t",
+            index_col=0,
+            names=[
+                base_name.replace("_counts.txt", ""),
```

Comment:
I think the expectation is for this to have a `_ref` suffix, to indicate the reference allele

Reply:
it will. The files from the previous step are named "[Replicate]_ref_counts.txt".